ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml

```diff
@@ -123,61 +123,77 @@ tests:
 - as: e2e-azure
   interval: 24h
   steps:
-    cluster_profile: azure4
+    cluster_profile: azure-2
+    env:
+      BASE_DOMAIN: ci2.azure.devcluster.openshift.com
     workflow: openshift-e2e-azure
 - as: e2e-azure-techpreview
   interval: 48h
   steps:
-    cluster_profile: azure4
+    cluster_profile: azure-2
+    env:
+      BASE_DOMAIN: ci2.azure.devcluster.openshift.com
     workflow: openshift-e2e-azure-techpreview
 - as: e2e-azure-compact
   interval: 48h
   steps:
-    cluster_profile: azure4
+    cluster_profile: azure-2
     env:
+      BASE_DOMAIN: ci2.azure.devcluster.openshift.com
```

Comment:
qq: is there a reason i don't see this env change being reflected on any job objects in the other file?

Reply:
This should be injected in the `install-config.yaml`

See: https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh#L26

Reply:
@abhinavdahiya If you mean the job yaml file, this env var doesn't have to, because it will be consumed by ci-operator itself.

Reply:
> @abhinavdahiya If you mean the job yaml file, this env var doesn't have to, because it will be consumed by ci-operator itself.

Ah, that makes sense then..